src/QmlControls/ParameterEditorDialog.qml

```diff
@@ -156,6 +156,15 @@ QGCViewDialog {
                     delegate : QGCCheckBox {
                         text : modelData
                         checked : fact.value & fact.bitmaskValues[index]
+                        function check_uncheck_all() {
+                            if (text.toLowerCase() == "all") {
```

Comment:
This is a bit too much of a special case to do without knowing this is correct from the meta data. "All" may not always mean this.